SlaveWAMPServer.js

```diff
@@ -145,7 +144,14 @@ class SlaveWAMPServer extends WAMPServer {
 			callback('RPC response timeout exceeded');
 			this.deleteCall(request);
 		}, this.internalRequestsTimeoutMs);
-		return setWith(this.interProcessRPC, `${request.socketId}.${request.procedure}.${request.signature}`, { callback, requestTimeout }, Object);
+
+		if (!this.interProcessRPC[request.socketId]) {
+			this.interProcessRPC[request.socketId] = {};
+		}
+		if (!this.interProcessRPC[request.socketId][request.procedure]) {
+			this.interProcessRPC[request.socketId][request.procedure] = {};
+		}
+		this.interProcessRPC[request.socketId][request.procedure][request.signature] = { callback, requestTimeout };
```

Comment:
If this deep structure is not changed maybe better use `Object.assign`?

Reply:
Thanks a lot, @willclarktech, I cannot use Object.assign - 1 procedure can have many calls assigned.

Reply:
got it